packages/nodes-base/nodes/Form/cssVariables.ts

```diff
@@ -0,0 +1,70 @@
+export const cssVariables = `
+:root {
+	--font-family: 'Open Sans', sans-serif;
+	--font-weight-normal: 400;
+	--font-weight-bold: 600;
+	--font-size-body: 12px;
+	--font-size-label: 14px;
+	--font-size-test-notice: 12px;
+	--font-size-input: 14px;
+	--font-size-header: 20px;
+	--font-size-paragraph: 14px;
+	--font-size-link: 12px;
+	--font-size-error: 12px;
+	--font-size-html-h1: 28px;
+	--font-size-html-h2: 20px;
```

Comment:
I don't see some of these variables (like this one) defined in the Form templates. Will these work?

Reply:
They're not defined in every form template just in `form-trigger.handlebars`. 
![CleanShot 2025-02-28 at 07 29 55](https://github.com/user-attachments/assets/ed896d90-68f6-475e-a5e6-e5ddc1c46832)